jtdiff/util/tests/TestTree.java

```diff
@@ -0,0 +1,80 @@
+package jtdiff.util.tests;
+
+
+
+
+
+import java.util.ArrayList;
+
+import org.junit.Before;
+
+
+import jtdiff.util.Tree;
+import jtdiff.util.TreeNode;
+import jtdiff.util.Visitor;
+
+class TraversalVisitor extends Visitor {
+	private ArrayList<String> traversal = new ArrayList<>();
+	
+	@Override
+	public void visit(TreeNode node) {
+		traversal.add(node.debugString());
+		
+	}
+	
+	public String[] getTraversal() {
+		return traversal.toArray(new String[traversal.size()]);
+	}
+	
+}
+
+
+/**
+ * 
+ * Tests the functionality of Tree
```

Comment:
This class does not test anything now. It is used to build trees to be used by its children